pyramid_swagger/ingest.py

```diff
@@ -167,12 +168,16 @@ def get_swagger_spec(settings):
     :rtype: :class:`bravado_core.spec.Spec`
     """
     schema_dir = settings.get('pyramid_swagger.schema_directory', 'api_docs/')
-    with open(os.path.join(schema_dir, 'swagger.json'), 'r') as f:
+    schema_path = os.path.join(schema_dir, 'swagger.json')
```

Comment:
The relevant ticket is Yelp/bravado-core#42. Do you want the comment in the code as well, that should not be necessary?